src/ValidationError.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace Schemer;
+
+class ValidationError
+{
+    private /* const */ $messages = [
```

Comment:
The only way I can think to do it is to define a translation set per validator that then gets concatenated with all the others at result time (and overwritten by user-defined stuff), which would give you some separation and decoupling. The only thing I'm uncomfortable with here is that validators are now tied to translations, and I guess the ideal would be the other way round, but I'm too hungry to conceptualise that at the moment...